### PR TITLE
[front] - feature(front): toggle navigation bar

### PR DESCRIPTION
**Pull request by JulesBelveze**
## Description

This PR aims at displaying a button hide/show the navigation bar on laptop: https://github.com/dust-tt/dust/issues/4988#issuecomment-2122616292

Screenshot 2024-05-22 at 16 00 49
Screenshot 2024-05-22 at 16 00 27
Screenshot 2024-05-22 at 16 00 40

## Risk

Breaking UI

## Deploy Plan

Deploy `front`

### CODE REVIEW

File: front/components/sparkle/AppLayout.tsx [hunk 1]
```diff
@@ -419,14 +435,21 @@ export default function AppLayout({
           >
             <div
               className={classNames(
-                "mx-auto h-full",
-                isWideMode ? "w-full" : "max-w-4xl px-6"
+                "mx-auto h-full w-full",
+                isWideMode ? "w-full" : "max-w-4xl"
               )}
```
Comment (JulesBelveze) [inline front/components/sparkle/AppLayout.tsx:440]: @Duncid Having trouble here do you have a lead? My current status is that 
I'm in a situation where fixing one breaks the other one.
Screenshot 2024-06-05 at 15 20 05
Screenshot 2024-06-05 at 15 19 00

**Comment by JulesBelveze** [comment]
@Duncid @flvndvd feel like this isn't the right way to achieve it. Do you have any recommendation?

**Comment by flvndvd** [comment]
> @Duncid @flvndvd feel like this isn't the right way to achieve it. Do you have any recommendation?

Let's spend 1 hour pair coding on it tomorrow (friday) afternoon @JulesBelveze?

**Comment by JulesBelveze** [comment]
Closing it due to recent refactoring.

**Comment by JulesBelveze** [comment]
New one https://github.com/dust-tt/dust/pull/5764

**Pull request closed without merging**